### ISSUE TITLE
Trades summary panel hardcoded USDT as currency

### ISSUE THREAD

**Issue by daniila** [froggleston/freqtrade-frogtrade9000#6]
The values seem correct, but the stake currency is displayed as USDT no matter what you use, BUSD in my case.

**Comment by froggleston** [comment]
Yes, this is because I'd have to read each config / run show_config and grab it from there. It's possible I suppose!
